src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -16,53 +16,59 @@
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageTags>Microsoft Azure WebJobs Durable Extension Orchestration Workflow Functions</PackageTags>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/Azure/azure-functions-durable-extension</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/Azure/azure-functions-durable-extension/</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <WarningsAsErrors />
-    <StyleCopTreatErrorsAsWarnings>true</StyleCopTreatErrorsAsWarnings>
-  </PropertyGroup>
-  
+
+  <!--
+    .NET Framework 4.x settings: These are specifically for Functions V1.
+  -->
   <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
-    <DefineConstants>FUNCTIONS_V1</DefineConstants>
+    <DefineConstants>$(DefineConstants);FUNCTIONS_V1</DefineConstants>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
-
-  <ItemGroup>
-    <!--Don't include Auth and Correlation classes in Function V1.-->
-    <Compile Include="**/*.cs" Exclude="Auth/*.cs;Correlation/*.cs;**/obj/**/*.cs" Condition="'$(TargetFramework)' == 'net462'" />
-  </ItemGroup>
-
   <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
+    <!-- Don't include Auth and Correlation classes in Function V1. -->
+    <Compile Include="**/*.cs" Exclude="Auth/*.cs;Correlation/*.cs;**/obj/**/*.cs" />
+    <!-- Don't increase below versions without significantly testing on Functions V1!
+         Increasing these versions increments some dependencies that have binding redirects in Functions V1. -->
     <PackageReference Include="Azure.Identity" Version="1.1.1" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.3.0" />
-
-    <!--Don't increase below versions without significantly testing on Functions V1. Increasing these versions increments
-    some dependencies that have binding redirects in Functions V1.-->
     <PackageReference Include="Microsoft.Extensions.Options" Version="1.0.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="1.0.2" />
     <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.4" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="1.0.4" />
   </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' != 'net462'">
+  
+  
+  <!-- 
+    .NET Standard 2.0 settings: These are for Functions V2 and above but may not contain some V3+ settings.
+    This is also the target for extension bundles V2 and V3.
+  -->
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Should we add a comment noting Functions V2 is deprecated and that therefore we're less inclined to update these dependencies?

Reply:
I don't know what comment I'd write here. There are still a lot of customers using Durable on Functions V2 and I think the decision about whether to make updates to these dependencies may need to be made on a case-by-case basis.

Reply:
Fair enough. I was just concerned about the duplication of dependency declarations across TFMs, but I guess there's nothing we can really do about that. We can leave this as is

Reply:
I originally considered removing the duplication, but I figured it's very possible that we may want to change the common dependencies across these two targets.  Having them be declared independently makes it easier for us to change one without accidentally changing the other (or otherwise adding complicated conditional logic).